doc/md/examples/Unpiped.mo

```diff
@@ -0,0 +1,7 @@
+import Iter "mo:base/Iter";
```

Comment:
why is this test needed?

Reply:
I wanted to test the examples in the doc, but didn't include them there because of the annoying imports.